gc/base/IndexableObjectScanner.hpp

```diff
@@ -79,19 +107,24 @@ class GC_IndexableObjectScanner : public GC_ObjectScanner
 	}
 
 	/**
-	 * Set up the scanner.
-	 * @param[in] env current environment (per thread)
+	 * Get the current element of the array
 	 */
-	MMINLINE void
-	initialize(MM_EnvironmentBase *env)
+	MMINLINE fomrobject_t *getScanPtr() { return _scanPtr; }
+
+	/**
+	 * Get the address of the next element in the array.
+	 * @return NULL if there are no more elements
+	 */
+	MMINLINE fomrobject_t *
+	next()
```

Comment:
I believe you said you'll rename this to nextIndexableElement

Reply:
I thought I had. I think I'm missing other changes too